org.munadi.Munadi.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "org.munadi.Munadi",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "munadi",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc", 
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-config/autostart:create",
```

Comment:
I am not sure if allowing the app to create an autostart file is a good idea as it should bypass sandbox this way. Does the app know how to handle this so it won't start without the sandbox when running in Flatpak?

Reply:
Yes, it has to go.

Reply:
@AsciiWolf well spotted, that's the safest way I found.

Reply:
At least until we get an appropriate portal

Reply:
@barthalion is my solution acceptable? Sandbox is still intact also ability to run at startup is critical for this app!
Thanks.

Reply:
Done

Reply:
Sorry, I was away. I disagree about reasoning. Users can add it to autostart manually in their DE themselves.

Reply:
@barthalion 
I'm sorry, I disagree with your disagreement.

Expecting the user to manually add it is horrible idea from UX prospective.
There is no built-in way to do it in gnome even! So limiting the workaround is not acceptable unless a better sane solution exists.

Thanks.